.codecov.yml

```diff
@@ -17,6 +17,15 @@ coverage:
         base: auto
     patch:
       default:
+        flags:
+          - asan
+          - integration
+          - integrationv2
+        target: auto
+        threshold: 0.5
```

Comment:
Do we want to allow for 50%  drop in coverage? 

 From Codecov docs: 
``` threshold: Allow the coverage to drop by X%, and posting a success status.```

Reply:
I wasn't sure if that meant 50% or .5%. We can address that in a separate change though since its what we already had